### PR TITLE
Adding DragonBoard support to SPI Display sample

### PR DESCRIPTION
**Pull request by apdutta**

### CODE REVIEW

**Comment by turkycat** [comment]
looks good

File: SPIDisplay/SPIDisplay/MainPage.xaml.cs [hunk 1]
```diff
@@ -34,11 +34,16 @@ public sealed partial class MainPage : Page
         //private const Int32 RESET_PIN = 4;                  /* We use GPIO 4 since it's conveniently near the SPI pins  */
 
         /* Uncomment for Raspberry Pi 2 */
-        private const string SPI_CONTROLLER_NAME = "SPI0";  /* For Raspberry Pi 2, use SPI0                             */
-        private const Int32 SPI_CHIP_SELECT_LINE = 0;       /* Line 0 maps to physical pin number 24 on the Rpi2        */
-        private const Int32 DATA_COMMAND_PIN = 22;          /* We use GPIO 22 since it's conveniently near the SPI pins */
-        private const Int32 RESET_PIN = 23;                 /* We use GPIO 23 since it's conveniently near the SPI pins */
-
+        //private const string SPI_CONTROLLER_NAME = "SPI0";  /* For Raspberry Pi 2, use SPI0                             */
```
Comment (bfjelds) [inline SPIDisplay/SPIDisplay/MainPage.xaml.cs]: why is SPI_CONTROLLER_NAME and SPI_CHIP_SELECT_LINE defined for each board?  they are always the same...I would suggest moving the identical settings to a section of constants that is not conditioned.

File: SPIDisplay/SPIDisplay/MainPage.xaml.cs [hunk 2]
```diff
@@ -34,11 +34,16 @@ public sealed partial class MainPage : Page
...
+        //private const Int32 SPI_CHIP_SELECT_LINE = 0;       /* Line 0 maps to physical pin number 24 on the Rpi2        */
+        //private const Int32 DATA_COMMAND_PIN = 22;          /* We use GPIO 22 since it's conveniently near the SPI pins */
+        //private const Int32 RESET_PIN = 23;                 /* We use GPIO 23 since it's conveniently near the SPI pins */
```
Comment (bfjelds) [inline SPIDisplay/SPIDisplay/MainPage.xaml.cs]: the rpi2 code was uncommented previously ... does the sample build without any of these uncommented?

File: SPIDisplay/SPIDisplay/MainPage.xaml.cs (hunk 1 above)
Comment (apdutta) [inline SPIDisplay/SPIDisplay/MainPage.xaml.cs]: It is the same for these 3 boards but this may change if there's any further addition.

**Pull request merged**